Spanner/src/Database.php

```diff
@@ -871,6 +872,10 @@ public function runTransaction(callable $operation, array $options = [])
         };
 
         $delayFn = function (\Exception $e) {
+            $defaultRetryDelay = ['seconds' => 0, 'nanos' => 0];
+            if ($e->getCode() === Code::INTERNAL && strpos($e->getMessage(), 'RST_STREAM') !== false) {
+                return $defaultRetryDelay;
```

Comment:
Nit but I'd rather just skip setting the variable since it's only used once 
```suggestion
                return ['seconds' => 0, 'nanos' => 0];
```